meilisearch/client.py

```diff
@@ -463,6 +463,25 @@ def cancel_tasks(self, parameters: dict[str, Any]) -> TaskInfo:
         """
         return cancel_tasks(self.config, parameters=parameters)
 
+    def delete_tasks(self, parameters: dict[str, Any]) -> TaskInfo:
```

Comment:
Using types in dynamic languages is complex, but I believe we at least can allow or not some parameters. WDYT?

https://github.com/meilisearch/meilisearch-ruby/pull/389/files#diff-34321aa6a7ffc59be7968ad4251433b3f6f30cdd222ed8d54572bf37f0000d7bR8-R13

I have to do that on the ruby side because of the camelCase/snake_case conversion, but it could be an excellent addition to the python SDK... (a issue could be created in this case)

Reply:
Oh, great idea it could be a really nice addition but I don't know how to do that in python, to be honest.
I will check it and create an issue about it.